solr/contrib/ltr/src/java/org/apache/solr/ltr/model/LambdaMARTModel.java

```diff
@@ -23,9 +23,8 @@
 
 import org.apache.lucene.index.LeafReaderContext;
 import org.apache.lucene.search.Explanation;
-import org.apache.solr.ltr.model.LTRScoringModel;
-import org.apache.solr.ltr.model.ModelException;
 import org.apache.solr.ltr.norm.Normalizer;
+import org.apache.solr.ltr.ranking.Feature;
```

Comment:
Moving Feature.java from ltr.ranking to ltr.feature would be a next separate pull request.